e2etest/bgp_test.go

```diff
@@ -687,12 +689,76 @@ var _ = ginkgo.Describe("BGP", func() {
 				pairExternalFRRWithNodes(cs, c, ipFamily)
 
 				for j := 0; j <= i; j++ {
-					validateFRRPeeredWithNodes(cs, frrContainers[j], ipFamily)
+					validateFRRPeeredWithNodes(cs, frrContainers[j], ipFamily, BeNil)
 				}
 			}
 		},
 			table.Entry("IPV4", "ipv4"),
 			table.Entry("IPV6", "ipv6"))
+
+		table.DescribeTable("configure bgp community NOPEER attribute and verify FRR session can't get to established state",
```

Comment:
I'd just add a test verifying that the community gets propagated to the other end.
I tried to tinker with FRR, one option is to run `show bgp community-info` and to check the output. In my case I got 
```
show bgp community-info 
Address Refcnt Community
[0x7f92505791d0] (1) no-advertise
```

So for example running it on the external containers before creating the service, check no communities are sent over, create the service and check there is a community.

Reply:
no adv is open type as I stated above I used no peer

Reply:
yes I am suggesting using something that gets propagated so we have more confidence.